### DISCUSSION TITLE
Is it possible to make multiple visualization objects (e.g. lines or paths) use the same coordinate system?

### DISCUSSION THREAD

**Discussion by KotobaTrilius** [typst/typst#4082] [Q&A]
I want to draw two paths, for example,

```
#path(
  (0pt, 0pt),
  (40pt, 50pt),
)

#path(
  (0pt, 0pt),
  (80pt, 20pt),
)
```

and I want them to share a same origin, so that they start from the same point.

It is possible to achieve this by manually moving their positions, but I wonder if there is a more convenient way.

**Comment by Enivex** [comment]
Cetz is the answer here. Using the built in functions directly is painful at the moment.
